scripts/fill_in_data.sql

```diff
@@ -0,0 +1,40 @@
+
+INSERT INTO public.products (product_id, title, description, price)
+VALUES('7567ec4b-b10c-48c5-9345-fc73c48a80aa', 'SPLAT Biocalcium Toothpaste', 'Biocalcium is a multiple-action toothpaste with bioactive calcium obtained from eggshell and hydroxyapatite, an essential component in the composition of dental enamel. The high content of these ingredients makes it possible to strengthen and remineralise dental enamel and reduce hypersensitivity.', 5);
```

Comment:
You don't have to insert ID of each product on your own, it is the responsibility of SQL to generate IDs